packages/user-interface/src/tests/mock/data/OverviewPageRequests.mock.tsx

```diff
@@ -0,0 +1,170 @@
+import { FetchResult, GraphQLRequest } from "@apollo/client";
+import { QUERY_GET_TAKEN, QUERY_GET_ZAKEN } from "@nl-portal/nl-portal-api";
+
+const getZaken: {
```

Comment:
Kunnen we niet een de mocks voor getZaken hergebruiken. Op dit moment doen we ze per "pagina" maar het is steeds dezelfde getZaken query toch?

Reply:
@Borai check, ga ik naar kijken.